contrib/babelfishpg_tsql/src/tsqlIface.cpp

```diff
@@ -2679,6 +2679,8 @@ rewriteBatchLevelStatement(
 				auto commas = cctx->COMMA();
 				GetTokenFunc<TSqlParser::Function_optionContext*> getToken = [](TSqlParser::Function_optionContext* o) { return o->SCHEMABINDING(); };
 				bool all_removed = removeTokenFromOptionList(expr, options, commas, ctx, getToken);
+				getToken = [](TSqlParser::Function_optionContext* o) { return o->execute_as_clause()->CALLER(); };
+				all_removed = removeTokenFromOptionList(expr, options, commas, ctx, getToken);
```

Comment:
You would also need to add the appropriate checks inside the following conditions 
1. `ctx->create_or_alter_function()->func_body_returns_table()`
2. `if (ctx->create_or_alter_function()->func_body_returns_scalar())`